man/systemd-sleep.conf.xml

```diff
@@ -160,8 +157,11 @@
         url="https://www.kernel.org/doc/html/latest/admin-guide/pm/sleep-states.html#basic-sysfs-interfaces-for-system-suspend-and-hibernation">the
         kernel documentation</ulink> for more details.</para>
 
-        <para>Note that hybrid sleep corresponds to the <literal>suspend</literal> disk mode. If <varname>HybridSleepMode=</varname>
-        is overridden, you might get plain hibernation instead.</para>
+        <para>
+        <citerefentry><refentrytitle>systemd-hybrid-sleep.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+        corresponds to the <literal>suspend</literal> disk mode. If that's not supported, it will fall back to
```

Comment:
I don't think a service can "correspond" to a suspend mode… "Implies", "uses", something else…?